corehq/apps/hqwebapp/static/hqwebapp/js/requirejs_config.js

```diff
@@ -12,6 +12,8 @@ requirejs.config({
         "datatables.bootstrap": "datatables-bootstrap3/BS3/assets/js/datatables",
         "datatables.scroller": "datatables-scroller/js/dataTables.scroller",
         "datatables.colReorder": "datatables-colreorder/js/dataTables.colReorder",
+        "jquery-mousewheel": "jquery-mousewheel/jquery.mousewheel",
```

Comment:
Does it not work to use `jquery-mousewheel/jquery.mousewheel` and `datetimepicker/build/jquery.datetimepicker.full.min` as the dependencies in js? I forget what exactly it is about datatables extensions that necessitated this config, but most of our third-party code doesn't need to be added here.

Reply:
it was complaining about this when I tried it (specifically with `jquery-mousewheel`), because somewhere else in datetimepicker it references `jquery-mousewheel` as the dependency...so it's definitely needed here. I figured if I was adding `jquery-mousewheel`, i might as well add `datetimepicker` since it was so long

Reply:
Cool, can we keep `mousewheel` here then but not the datetimepicker? I'd like "dependency names match filenames" to be as firm a rule as possible for simplicity's sake, even if it means typing out long names.

Reply:
sure, done!

Reply:
thanks!